main/cropper.js

```diff
@@ -16,48 +16,98 @@ const prodPath = formatUrl({
 
 const url = isDev ? devPath : prodPath;
 
-let cropper = null;
+const croppers = {};
```

Comment:
A `Map` would be more appropriate here, especially since the display IDs are numbers and object keys can only be strings, so they're coerced to strings and that can create weird situations.